src/utils/consts/sideBar.consts.ts

```diff
@@ -0,0 +1,31 @@
+export const sideBarData = [
```

Comment:
const should be uppercased so SIDE_BAR_DATA

Reply:
aha , dobrze że wcześniej o tym wspomniałeś